proto/attestation/reference_value.proto

```diff
@@ -26,11 +26,11 @@ option java_multiple_files = true;
 option java_package = "com.google.oak.attestation.v1";
 
 message ReferenceValues {
-  RootReferenceValues root_layer = 1;        // Layer 0
+  RootLayerReferenceValues root_layer = 1;   // Layer 0
   repeated LayerReferenceValues layers = 2;  // Layers 1 through N.
 }
 
-message RootReferenceValues {
+message RootLayerReferenceValues {
```

Comment:
The naming seems inconsistent between `${THING}ReferenceValue` and `Verify${THING}`. Or are they semantically distinct?